src/briefcase/platforms/linux/system.py

```diff
@@ -1295,6 +1304,22 @@ class LinuxSystemPublishCommand(LinuxSystemMixin, PublishCommand):
     description = "Publish a Linux system project."
 
 
+def find_changelog_filename(base_path):
```

Comment:
Two comments:
1. This method should have a docstring, describing what it does
2. Changelogs are a cross-platform feature, so this discovery mechanism should be in a cross-platform location. Since it's could be used by the `convert` command, I'd suggest that as a better location for defining this method.